lms/djangoapps/verify_student/views.py

```diff
@@ -1330,4 +1329,12 @@ def _redirect_no_initial_verification(self, user, course_key):
             u"for the course %s.",
             user.id, course_key
         )
-        return redirect(reverse('verify_student_verify_later', kwargs={'course_id': unicode(course_key)}))
+        return redirect(reverse('verify_student_verify_now', kwargs={'course_id': unicode(course_key)}))
+
+
+class VerifyLaterView(RedirectView):
```

Comment:
please add a comment that this should be removed after release (unless that's not actually the plan...)